core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/template/ResponsiveContent.kt

```diff
@@ -87,7 +85,7 @@ private fun ExpandedContent(
             ) {
                 Surface(
                     modifier = Modifier.requiredWidth(WindowSizeClass.MEDIUM_MAX_WIDTH.dp),
-                    tonalElevation = MainTheme.elevations.raised,
+                    tonalElevation = MainTheme.elevations.level1,
```

Comment:
Side note: I think we'd make our lives much easier if `ResponsiveContent` didn't modify the surface color via `tonalElevation`. We could use `shadowElevation` or simply restrict the content size and use no surface and/or elevation at all.

Reply:
Material 3 is not using shadows to indicate depth and hierarchy anymore. It emphasizes using tonal elevation instead. The Material 3 API is still exposing this for compatibility reasons and I opted to not expose shadows to avoid confusion.